src/app/global/local_app_settings.nim

```diff
@@ -15,7 +15,7 @@ const DEFAULT_SCROLL_VELOCITY = 0 # unset
 const DEFAULT_SCROLL_DECELERATION = 0 # unset
 const LAS_KEY_CUSTOM_MOUSE_SCROLLING_ENABLED = "global/custom_mouse_scroll_enabled"
 const DEFAULT_CUSTOM_MOUSE_SCROLLING_ENABLED = false
-const DEFAULT_VISIBILITY = 2 #windowed visibility, from qml
+const DEFAULT_VISIBILITY = 5 #full screen visibility, from qml
```

Comment:
Almost sure this will result, at least on Linux, in the window not having any decoration at all

Reply:
That was my first intention. But unfortunately it doesn't work. It's still windowed.

What about defaulting to maximized then? It should still have the window decorations 🤔

Reply:
> That was my first intention. But unfortunately it doesn't work. It's still windowed.
> 
> What about defaulting to maximized then? It should still have the window decorations 🤔

Yeah that's a better idea :) Or you could make the `DEFAULT_VISIBILITY` conditional based on the runtime OS

Reply:
yeah, I think I'll do that. Maximized is not going to work on all configurations and screen sizes.

Reply:
Tested on iOS and with `const DEFAULT_VISIBILITY = 1` I can state that it does not fit the requirement. I agree also in making it conditional